gradio/blocks.py

```diff
@@ -1453,21 +1477,7 @@ def reverse(text):
         if not self.enable_queue and self.progress_tracking:
             raise ValueError("Progress tracking requires queuing to be enabled.")
```

Comment:
We should move this logic into `validate_queue_settings()` so that it is also checked when the gradio app is mounted.

Reply:
Good idea! Done